### PR TITLE
[Foundation] Set a middle align for table cells

### PR DESCRIPTION
**Pull request by floriank**
Meets https://community.openproject.org/work_packages/19216

### CODE REVIEW

**Comment by TeatroIO** [comment]
I've prepared a stage to preview changes. <a href='http://floriank-fix-19126-table-middle-align-attributes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com' target='_blank'>Open stage</a> or <a href='http://floriank-fix-19126-table-middle-align-attributes.opf-openproject-2af22bf12c7dfdad4ddf.ttrcloud.com/teatro.log' target='_blank'>view logs</a>.

File: app/assets/stylesheets/content/_tables.sass [hunk 1]
```diff
@@ -54,6 +52,8 @@ table
         text-align: right
         a
           padding-right: 0.6em
+      .form--text-field, .form--select
```
Comment (NobodysNightmare) [inline app/assets/stylesheets/content/_tables.sass]: I think you also want to apply this to text areas:
lower margin wrong

Inspecting the elements you can see that a text area still has 16px margin at the bottom, I guess this is the one you are explicitly removing here for other elements.

_Note: If you wonder how, just add a custom field of type `Long Text`_

Comment (floriank) [inline app/assets/stylesheets/content/_tables.sass]: i like how all the names are self explanatory and easy to remember :santa:

**Comment by NobodysNightmare** [comment]
One remark (see above).

Looks good otherwise :+1:

**Comment by floriank** [comment]
screenshot from 2015-03-19 17 37 46

Should look better now

**Pull request merged**